.github/workflows/ci-iac.yml

```diff
@@ -111,31 +107,31 @@ jobs:
           aws-region: us-west-2
 
       - name: Terraform Setup
-        uses: hashicorp/setup-terraform@v3
+        uses: opentofu/setup-opentofu@v1
         with:
-          terraform_version: ${{ env.tf_version }}
+          tofu_version: ${{ env.tf_version }}
 
       - name: Terraform Init
-        working-directory: ${{ matrix.env.tf_working_dir }}
-        run: terraform init
+        working-directory: terraform/app
+        run: tofu init -backend-config=${{ matrix.env.environment_name }}.s3.tfbackend
 
```

Comment:
Missed a spot. This should've been changed to `-var-file` in 80425243a52a4f4e3e130a3c4625c2343b6d7c55.